index.html

```diff
@@ -2532,183 +2532,169 @@ <h1>Servient Implementation</h1>
+                The portability can also foster <a>Thing</a> customization by installing behavior following the app concept known from smartphones.
             </p>
         </section>
         <section>
-            <h2>Exposed Thing and Consumed Thing Abstractions</h2>
-            <p> The <a>WoT Runtime</a> and <a>WoT Scripting API</a> provide
-                <a>Exposed Thing</a> abstractions that represent Things exposed by
-                the <a>Servient</a> (network interfaces presented to the outside when the
-                Servient is acting as a server) and <a>Consumed Thing</a>
-                abstractions representing other Things the Servient can
-                communicate with in a client role.
-            </p>
-            <p> In the <a>WoT Scripting API</a> these abstractions are
-                represented by actual object instances.
-                These objects provide an abstract interface at the
-                level of interactions, not specific protocols. 
-                The <a>WoT Runtime</a> manages the translation of 
-                these higher-level interactions to
-                concrete protocol transactions using the Protocol Binding.
+            <h2>WoT Runtime</h2>
+            <p>
+                Technically, the <a>Thing</a> abstraction and its <a>Interaction Model</a> is implemented in a runtime system.
+                This <a>WoT Runtime</a> maintains the execution environment for the behavior implementation and is able to expose and/or consume <a>Things</a>,
+                and hence must be able to process, serialize, serve, and fetch <a>WoT Thing Descriptions</a>.
+            </p>
+            <p>
+                A <a>WoT Runtime</a> needs to provide certain operations to manage the lifecycle of <a>Things</a>,
+                or more precicely their software abstractions and descriptions.
+                The details of such operations vary among different implementations.
+                A lifecycle management (LCM) system may encapsulate those lifecycle operations within a <a>Servient</a>
+                and use internal interfaces to realize the lifecycle management.
+                The <a>WoT Scripting API</a> includes one possible implementation of the LCM functionality.
+            </p>
+            <p>
+                Southbound, the <a>WoT Runtime</a> interfaces with the protocol stack implementation of the <a>Servient</a>
+                and decouples the behavior implementation from the details of the <a>Protocol Bindings</a>.
+                The <a>WoT Runtime</a> usually also interfaces with the underlying system,
+                for instance, to access local hardware or even proprietary communication protocols.
+                Both interfaces are implementation-specific, but must follow the <a>Thing</a> abstraction,
+                so that the behavior implementation need not depent on system-specific API calls.
```

Comment:
I always stumble over the term "behavior implementation".

Please consider rewording to: 
- "behavior" implementation 
or
- implementation of the behavior

Reply:
You mean putting the word behavior in quotes?

I don't understand why this issue only arises now and not when the term was introduced and caused big ripples throughout the document, nor when I tried to discuss if "behavior" is the term we want use.